go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go/storage v1.8.0
+	github.com/Azure/azure-sdk-for-go v42.3.0+incompatible
```

Comment:
is this change related?  to verify run "go mod tidy" before the PR

Reply:
Yes, I use `ioext.CopyFile()` from this module. We'd had an indirect dependency to this module, so I used it.